nml/actions/action0.py

```diff
@@ -900,22 +972,22 @@ def get_language_translation_tables(lang):
         return [action0]
     return []
 
+
 disable_info = {
     # Vehicles: set climates_available to 0
-    0x00 : {'num': 116, 'props': [{'num': 0x06, 'size': 1, 'value': 0}]},
-    0x01 : {'num':  88, 'props': [{'num': 0x06, 'size': 1, 'value': 0}]},
-    0x02 : {'num':  11, 'props': [{'num': 0x06, 'size': 1, 'value': 0}]},
-    0x03 : {'num':  41, 'props': [{'num': 0x06, 'size': 1, 'value': 0}]},
-
+    0x00: {"num": 116, "props": [{"num": 0x06, "size": 1, "value": 0}]},
+    0x01: {"num": 88, "props": [{"num": 0x06, "size": 1, "value": 0}]},
+    0x02: {"num": 11, "props": [{"num": 0x06, "size": 1, "value": 0}]},
+    0x03: {"num": 41, "props": [{"num": 0x06, "size": 1, "value": 0}]},
```

Comment:
These kind of formatting changes are not helpful when the alignment is for purpose to better identify / compare lines